core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -1022,9 +1088,10 @@ ExWorkProcRetcode ExHbaseAccessUpsertVsbbSQTcb::work()
 	    if (NOT hbaseAccessTdb().computeRowsAffected())
 	      matches_ = 0;
 
-	    if ((step_ == DONE) &&
+             if ((step_ == DONE) &&
```

Comment:
Was this indentation change intentional? It now looks like the "if ((step_ == DONE)..." is executed when "NOT hbaseAccessTdb().computeRowsAffected" is true.